setup.cfg

```diff
@@ -0,0 +1,50 @@
+[metadata]
+name = strava_py
+version = 0.0.1
+description = Create artistic visualisations with your exercise data
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/marcusvolz/strava_py
+author = Marcus Volz
+license = MIT
+license_file = LICENSE
+classifiers =
+    Development Status :: 3 - Alpha
+    License :: OSI Approved :: MIT License
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Topic :: Artistic Software
+    Topic :: Multimedia :: Graphics
+    Topic :: Scientific/Engineering :: Visualization
+keywords =
+    strava
+    artistic visualisations
+    artistic
+    visualisation
+    exercise data
+    exercise
+project_urls =
+    Source=https://github.com/marcusvolz/strava_py
+
+[options]
+packages = find:
+install_requires =
+    fit2gpx
+    gpxpy
+    matplotlib
+    pandas
+    seaborn
+python_requires = >=3.7
+package_dir = =src
+zip_safe = True
+
+[options.packages.find]
+where = src
+
+[options.entry_points]
+console_scripts =
+    strava_py = strava_py.cli:main
```

Comment:
And this `strava_py` is the name for the CLI, what the user runs on the command line. Doesn't need to be the same as `name=` above, but less confusing if they are the same.